openml/testing.py

```diff
@@ -119,5 +120,58 @@ def _check_dataset(self, dataset):
         self.assertIn(dataset['status'], ['in_preparation', 'active',
                                           'deactivated'])
 
+    def _check_fold_evaluations(
```

Comment:
rename to `_check_fold_timing_evaluations`?

Reply:
Yes!